README.md

```diff
@@ -16,7 +16,7 @@ from the rest of your system by vendorizing its dependencies. This is the
 recommended way of installing `poetry`.
 
 ```bash
-curl -sSL https://raw.githubusercontent.com/sdispater/poetry/master/get-poetry.py | python
+curl -sSL https://raw.githubusercontent.com/sdispater/poetry/master/get-poetry.py | sudo -H python3
```

Comment:
This introduces two changes: `python` -> `python3`; and the introduction of `sudo -H`.

On my machine (macOS with homebrew-installed python@2 and python3), using this with just the `python3` change had the desired affect: it created a python3 virtual environment such that both 
`poetry run which python` and `poetry run which python3` refer to a Python in the appropriatevirtualenv,  and both `poetry run python --version` and `poetry run python3 --version` display `Python 3.6.5`.

So yay for that! (I'd been trying to figure out how to use Poetry for Python 3 development, and was surprised that putting `python = ">=3.0"` in `pyproject.toml` didn't work. This PR saved me.)

I suspect, based on prior experience with `sudo` and homebrew, that if I'd run it with the `sudo`, I would have mixed up some ownerships in  `/usr/local/` in a way that could cause trouble downstream.

Maybe the `sudo -H` part of this change can either simply be removed, or moved to a different PR since if it's necessary at all, its has different compatibility issues and may be solving a different issue.